src/plugins/console/public/application/hooks/use_send_current_request/send_request.ts

```diff
@@ -128,9 +128,9 @@ export function sendRequest(args: RequestArgs): Promise<RequestResult[]> {
       } catch (error) {
         let value;
         const { response, body } = error as IHttpFetchError;
-        const contentType = response?.headers.get('Content-Type') ?? '';
+        const contentType = (response?.headers.get('Content-Type') as BaseResponseType) ?? '';
```

Comment:
I see two places in that file where we do `as BaseResponseType`. Could you please extract this one into separate method `getContentType`